src/status_im/contexts/wallet/send/input_amount/style.cljs

```diff
@@ -38,3 +39,22 @@
   {:height      40
    :width       "100%"
    :align-items :center})
+
+(def token-not-available-container
+  {:height            90
+   :flex-direction    :row
+   :background-color  colors/danger-opa-5
+   :border-color      colors/danger-opa-10
+   :border-width      1
+   :border-radius     12
+   :margin-horizontal 20
+   :padding           12})
+
+(def token-not-available-content-container
+  {:margin-left 8
+   :align-items :flex-start})
+
+(def token-not-available-text
+  {:height 36
+   :flex   1
+   :color  colors/danger-50})
```

Comment:
please update this to use `(colors/resolve-color :danger theme 50)`. Imo it's better that we abstract the colors resolving to a system so we can ensure it follows the same mechanism in most cases and allows for easier future updates.

Reply:
Hmmm actually we need to use `danger-50` color which is a solid variant of danger family colors, resolve-color would just apply opacity to danger color.

Reply:
Whoops leave out the opacity var and this is correct 👍

Reply:
ie (colors/resolve-color :danger theme)

Reply:
Now updated it to use `(colors/resolve-color :danger theme)` 👍